src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form__big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label_big">
+        <input
+          type="text"
+          data-qa="keypress"
+          class="search-form__input search-form__input_big"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form__small"
+      data-qa="small"
```

Comment:
HTML Formatting: The attributes of the `<form>` element should each start on a new line with 2-space indentation, and the closing bracket should be aligned with the opening tag. This will improve readability and maintain consistency with proper HTML formatting.